Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp

```diff
@@ -264,6 +273,9 @@ void GeneralWidget::AddDescriptions()
   static const char* TR_HIDE_MOUSE_CURSOR_DESCRIPTION =
       QT_TR_NOOP("Hides the mouse cursor if it's on top of the emulation window.\n\nIf unsure, "
                  "leave this unchecked.");
+  static const char* TR_GRAB_MOUSE_CURSOR_DESCRIPTION =
+      QT_TR_NOOP("Grabs the mouse cursor if the emulation window is in focus.\n\nIf unsure, "
```

Comment:
I don't think this is a clear enough explanation for everyone. We need something more explanatory than the word "grabs".

Reply:
Sure. Got any ideas?

Reply:
Maybe something like "Prevents the mouse cursor from leaving the emulation windows when it is in focus."

Reply:
Well, it actually doesn't do that: It simply makes you unable to click on anything while the cursor is in focus. (Mostly useful when using an emulated Wiimote in windowed Mode)

Reply:
Oh, okay. Well, I guess that goes to show why the description could be more detailed :)

Reply:
"Prevents issues with accidentally clicking outside the emulation window" ?

Reply:
Fine by me. Needs @MayImilae's approval though.